client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -0,0 +1,144 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { StripeHookProvider } from '@automattic/calypso-stripe';
+
+/**
+ * Internal Dependencies
+ */
+import CreditCardForm from 'calypso/blocks/credit-card-form';
+import HeaderCake from 'calypso/components/header-cake';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import QueryStoredCards from 'calypso/components/data/query-stored-cards';
+import QueryUserPurchases from 'calypso/components/data/query-user-purchases';
+import titles from 'calypso/me/purchases/titles';
+import TrackPurchasePageView from 'calypso/me/purchases/track-purchase-page-view';
+import { clearPurchases } from 'calypso/state/purchases/actions';
+import { createCardToken, getStripeConfiguration } from 'calypso/lib/store-transactions';
+import {
+	getByPurchaseId,
+	hasLoadedUserPurchasesFromServer,
+} from 'calypso/state/purchases/selectors';
+import { getCurrentUserId, getCurrentUserLocale } from 'calypso/state/current-user/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import {
+	getStoredCardById,
+	hasLoadedStoredCardsFromServer,
+} from 'calypso/state/stored-cards/selectors';
+import { isRequestingSites } from 'calypso/state/sites/selectors';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import Layout from 'calypso/components/layout';
+import Column from 'calypso/components/layout/column';
+import PaymentMethodSidebar from 'calypso/me/purchases/components/payment-method-sidebar';
+import PaymentMethodLoader from 'calypso/me/purchases/components/payment-method-loader';
+
+function EditPaymentMethod( props ) {
```

Comment:
This component should probably be renamed to match the filename.

Reply:
Thanks for catching that. I've updated it.